### DISCUSSION TITLE
OAuth support?

### DISCUSSION THREAD

**Discussion by sic-guno** [TomorrowIdeas/plaid-sdk-php#37] [Q&A] [answered]
I was tasked with making sure we are supporting OAuth flows for Plaid and was wondering if this Library supports them. It looks like Token's create method supports the redirect_url parameter and the last release was after the OAuth upgrades on Plaid's side.

Thanks

**Comment by elliotkoss** [comment, marked as answer]
I upgraded yesterday. As long as you’re using a Link Token, all you need is
the redirect URI plus some JQuery updates to the second Link
initialization. And you can test in the sandbox with Platypus OAuth bank
(something like that).

Good luck.

Best,
Elliot

On Mon, May 17, 2021 at 1:12 PM sic-guno ***@***.***> wrote:

> I was tasked with making sure we are supporting OAuth flows for Plaid and
> was wondering if this Library supports them. It looks like Token's create
> method supports the redirect_url parameter and the last release was after
> the OAuth upgrades on Plaid's side.
>
> Thanks
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/TomorrowIdeas/plaid-sdk-php/discussions/37>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABJPFYLNA3A2WIBHNY55BHLTOFMBBANCNFSM45A6PU2Q>
> .
>
-- 
Best,
Elliot

**Reply by sic-guno** [reply]
Thanks for this amazing library Elliot!

**Comment by elliotkoss** [comment]
Oh, I didn’t create this great SDK. I just got the OAuth to work without
updating.

On Tue, May 18, 2021 at 12:21 PM sic-guno ***@***.***> wrote:

> Thanks for this amazing library Elliot!
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/TomorrowIdeas/plaid-sdk-php/discussions/37#discussioncomment-754127>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABJPFYJKJ7AL32FPTSRZVATTOKO2NANCNFSM45A6PU2Q>
> .
>
-- 
Best,
Elliot
